ticc.php

```diff
@@ -43,7 +43,10 @@
 
 
 EOT;
-    (new \ticc\Ticc ($argv))->run();
+
+    $ticc = new \ticc\Ticc ($argv);
+    $verbose_p = $ticc->get_parameter('verbose');
```

Comment:
I don't like the idea of pulling parameters out of the Ticc object to make decisions here, but this is entirely my fault for putting the error handling outside the place where parameters are available... This probably calls for a refactor of some sort, simplifying the main invocation logic here and splitting Ticc in to two or more lighter classes. I was already thinking of some sort of output handler class, and now you've got me thinking of an input handler to take over the args/parameters as well. (Side note: really not thrilled with the getopts library I used, if you have a better idea on that please jump in!)

In the mean time, this is great for now though!